modules/veteran_verification/METADATA.yml

```diff
@@ -0,0 +1,81 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Veteran Verification
+  description: |
+
+    The Veteran Verification API allows you to confirm the Veteran status for a given person, or retrieve a Veteran's service history or disability rating.
+
+    When confirming Veteran Status, the Status endpoint will return "Confirmed" or "Not Confirmed".
+
+    ## Quickstart Guide
+
+    ### 1. Get Access Credentials
+
+    Get started by filling out the form on the [Apply for VA Lighthouse Developer Access](/apply) page. 
+    
+    After submitting a request, you will receive your credentials for using the API in the Development environment, which allows you to try it out with mock data before moving to the Production environment.
+    
+    ### 2. Test the  API
+    
+    In the endpoint documentation below, we've provided a curl command builder for trying out the API before implementation with your app.
+
+    ### 3. Build your app
+
+    The base URI for the Veteran Verification API in the Development environment is:
+
+    dev-api.va.gov/services/veteran_verification/v0
+
+    Check out some of our sample apps located on our [GitHub page](https://github.com/department-of-veterans-affairs/vets-api-clients/tree/master/samples/). Please visit our VA Lighthouse [Support portal](/support) should you need further assistance.
```

Comment:
We can probably just link this as `[sample apps](linkgoeshere)` instead of specifying Github.